Sources/SwifterSwift/UIKit/UIButtonExtensions.swift

```diff
@@ -204,7 +204,28 @@ public extension UIButton {
             contentEdgeInsets = UIEdgeInsets(top: 0, left: insetAmount, bottom: 0, right: insetAmount)
         }
     }
-
+    
+    /// SwifterSwift: Put image to the right side of title
+    /// 
+    /// - Parameter spacing: spacing between title text and image.
+    func rightAlignImage(spacing: CGFloat) {
```

Comment:
I think this could be complemented with a `leftAlignImage` method ... maybe? What you think ?

Reply:
Hi @LucianoPAlmeida, I don't think so. `leftAlignImage` is the default behaviour of UIButton, right? Do we need to create an extension but not doing much? Or overriding the default behaviour?

Reply:
Makes sense :)